src/app/components/Image/index.tsx

```diff
@@ -155,16 +157,17 @@ const Image = ({
           </>
         ) : (
           <ImageWrapper>
-            {hasFallback && pageType === FRONT_PAGE && (
-              <>
-                <source srcSet={srcSet} type={mediaType} sizes={sizes} />
-                <source
-                  srcSet={fallbackSrcSet}
-                  type={fallbackMediaType}
-                  sizes={sizes}
-                />
-              </>
-            )}
+            {hasFallback &&
+              (pageType === FRONT_PAGE || pageType === HOME_PAGE) && (
+                <>
+                  <source srcSet={srcSet} type={mediaType} sizes={sizes} />
+                  <source
+                    srcSet={fallbackSrcSet}
+                    type={fallbackMediaType}
+                    sizes={sizes}
+                  />
+                </>
+              )}
```

Comment:
I'd have expected these changes to have caused tests fail for the image component? Can we add some tests to cover the differences in output between Home and Front pages and all other page types?

Reply:
I have a test fail on thee Image unit tests when I run them on my computer:

' Image - Canonical › should render the primary srcset and sizes

    expect(element).toHaveAttribute("srcset", "/test-image-200.jpg 200w, /test-image-500.jpg 500w") // element.getAttribute("srcset") === "/test-image-200.jpg 200w, /test-image-500.jpg 500w"

    Expected the element to have attribute:
      srcset="/test-image-200.jpg 200w, /test-image-500.jpg 500w"
    Received:
      srcset="/test-image-200.webp 200w, /test-image-500.webp 500w"'

which makes sense. But this error is not there in the github checks unit test failures. There are other errors going on, but I only see this one when I manually run it on my computer.

Reply:
<img width="553" alt="Screenshot 2024-05-02 at 12 57 51" src="https://github.com/bbc/simorgh/assets/54575620/30f1c73b-e536-48bd-837e-d139c4913a3d">

Should this one be failing?